lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
I had an off-github exchange from @ichuang and came to the conclusion that, from a business perspective, edX certificates are not compatible with the POC/CCX concept. 

@pwilkins -- you and I can follow up with the 15.390.1x course team. 

@ormsbee -- is it enough for us to say that certificates are not supported, or do we need to something else for this PR?